WordPress/WordPressTest/EUUSCompliance/CompliancePopoverViewModelTests.swift

```diff
@@ -0,0 +1,87 @@
+import XCTest
+@testable import WordPress
+
+final class CompliancePopoverViewModelTests: CoreDataTestCase {
+    let testDefaults = UserDefaults(suiteName: "compliance-popover-view-model-tests")
+
+    override func setUp() {
+        super.setUp()
+        testDefaults?.removeObject(forKey: UserDefaults.didShowCompliancePopupKey)
+
+        let windowManager = WindowManager(window: UIWindow())
+        WordPressAuthenticationManager(
+            windowManager: windowManager,
+            remoteFeaturesStore: RemoteFeatureFlagStore()
+        ).initializeWordPressAuthenticator()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        testDefaults?.removeObject(forKey: UserDefaults.didShowCompliancePopupKey)
```

Comment:
Nitpick. When tearing down, `super` should be called last, so that first we do any state-reset necessary for this subclass, and only then we let the super class do its, presumably more generic, work.

Reply:
Hmm interesting. Do you have a reference to this? I thought supers of these functions are no-op. I would to see the docs or something if sometihng like this is documented by Apple.

Reply:
@mokagio I see you're AFK. We'll merge the PR for now since we need to wrap this up. Please let us know your thoughts, and we can address this, if needed, in another PR.